python/pyiceberg/table/metadata.py

```diff
@@ -195,27 +196,21 @@ class TableMetadataV1(TableMetadataCommonFields, IcebergBaseModel):
     # When writing, we should stick to the same version that it was,
     # because bumping the version should be an explicit operation that is up
     # to the owner of the table.
-
     @root_validator(pre=True)
     def set_v2_compatible_defaults(cls, data: Dict[str, Any]) -> Dict[str, Any]:
         """Sets default values to be compatible with the format v2
-
```

Comment:
Can you revert some of the unnecessary whitespace changes?

Reply:
Not sure why this happened.  This should be caught by the linter